test/public-stats.test.js

```diff
@@ -467,7 +468,7 @@ describe('public-stats', () => {
           'SELECT day::TEXT, tested, retrieval_majority_found, retrievable FROM daily_deals'
         )
         assert.deepStrictEqual(created, [
-          { day: today, tested: 4, retrieval_majority_found: 3, retrievable: 1 }
+          { day: today, tested: 4, retrieval_majority_found: 2, retrievable: 1 }
```

Comment:
This change was needed after I fixed the bug described in https://github.com/filecoin-station/spark-evaluate/pull/336#discussion_r1749920681